gluster_exporter/main.go

```diff
@@ -32,17 +42,64 @@ func registerMetric(name string, fn func(), intervalSeconds int64) {
 	glusterMetrics = append(glusterMetrics, glusterMetric{name: name, fn: fn, intervalSeconds: time.Duration(intervalSeconds)})
 }
 
-func getPeerID() string {
-	return *peerid
+func getGlusterdWorkdir() (string, string) {
```

Comment:
Let this function only return workdir only. this function will be useful for others too

Reply:
uuid file logic can be handled in parser function itself